packages/browser-destinations/src/destinations/hubspot-web/index.ts

```diff
@@ -0,0 +1,62 @@
+import type { Settings } from './generated-types'
+import type { BrowserDestinationDefinition } from '../../lib/browser-destinations'
+import { browserDestination } from '../../runtime/shim'
+
+import trackCustomBehavioralEvent from './trackCustomBehavioralEvent'
+
+import trackPageView from './trackPageView'
+
+import upsertContact from './upsertContact'
+import { Hubspot } from './types'
```

Comment:
Nit: I think we generally prefer `import type` where appropriate, but not a big deal.

```suggestion
import type { Hubspot } from './types'
```